src/styles/components/molecules/Article.scss

```
@@ -1,13 +1,16 @@
 .article {
-  width: 440px;
+  position: relative;
+  min-width: 340px;
+  width: auto;
   & h4 {
     margin-bottom: 8px;
   }
   &-author {
     font-size: 24px;
-    margin-bottom: 9px;
+    margin-bottom: 9px; 
   }
   &-body {
+    width: 80%;
```

```suggestion
    width: 80%;
```

This is not necesary